bool active
bool failure
bool moving
float32 total_duration
float32 distance_to_robot
geometry_msgs/Point position
